includes/functions/functions_search.php

```diff
@@ -98,39 +98,45 @@ function zen_parse_search_string($search_str = '', &$objects = array()) {
 // Move on to the next word
                     $k++;
                     continue;
-                } else {
-                    /* If the $piece ends in double quotes, strip the double quotes, tack the
-                       $piece onto the tail of the string, push the $tmpstring onto the $haves,
-                       kill the $tmpstring, turn the $flag "off", and return.
-                    */
-                    $tmpstring .= ' ' . trim(preg_replace('/"/', ' ', $pieces[$k]));
+                }
+                /* If the $piece ends in double quotes, strip the double quotes, tack the
+                   $piece onto the tail of the string, push the $tmpstring onto the $haves,
+                   kill the $tmpstring, turn the $flag "off", and return.
+                */
+                $tmpstring .= ' ' . trim(preg_replace('/"/', ' ', $pieces[$k]));
 
 // Push the $tmpstring onto the array of stuff to search for
-                    $objects[] = trim($tmpstring);
+                $objects[] = trim($tmpstring); // Why does this need to again be trimmed when everything else has been so far?
```

Comment:
Line 1065 above prepends a space to the trimmed string. This trim here would remove that prepended space.

Reply:
Ditto.